src/OrchardCore/OrchardCore/Shell/ShellSettingsManager.cs

```diff
@@ -304,6 +299,15 @@ private async Task EnsureConfigurationAsync()
             _configuration = configuration;
         }
 
+        private async Task ReloadTenantsSettingsAsync()
+        {
+            using var disposable = _tenantsSettingsRoot as IDisposable;
```

Comment:
Where did you learn that from! This is C# magic

Reply:
I don't remember ;) I know that `var disposable` will be disposed without failing if it is null.